### PR TITLE
ENG-1344 SVOD Enrichment Atlas Model Changes part 3 

### PR DESCRIPTION
**Pull request by pawel-mtbcst**

### CODE REVIEW

File: src/main/java/org/atlasapi/persistence/media/entity/ProductionCompanyTranslator.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        return dbObject;
+    }
+
+    public ProductionCompany fromDBObjects(DBObject dbObject) {
+        return fromDBObject(dbObject, new ProductionCompany());
+    }
+
+    public ProductionCompany fromDBObject(DBObject dbObject) {
+        return fromDBObject(dbObject, new ProductionCompany());
+    }
```
Comment (EmiEnache) [inline src/main/java/org/atlasapi/persistence/media/entity/ProductionCompanyTranslator.java:33]: Are they both needed? What's the difference between these two?

File: src/main/java/org/atlasapi/persistence/media/entity/ProductionCompanyTranslator.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import com.mongodb.BasicDBObject;
+import com.mongodb.DBObject;
+import org.atlasapi.media.entity.common.Company;
+import org.atlasapi.media.entity.common.ProductionCompany;
+import org.atlasapi.persistence.ModelTranslator;
+
+import javax.annotation.Nullable;
+
+public class ProductionCompanyTranslator implements ModelTranslator<ProductionCompany> {
+
```
Comment (EmiEnache) [inline src/main/java/org/atlasapi/persistence/media/entity/ProductionCompanyTranslator.java:13]: `public static final String COMPANY = "company";`

File: src/main/java/org/atlasapi/persistence/media/entity/ProductionCompanyTranslator.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    private final CompanyTranslator companyTranslator = new CompanyTranslator();
+
+    public DBObject toDBObject(ProductionCompany model) {
+        return toDBObject(new BasicDBObject(), model);
+    }
+
+    @Override
+    public DBObject toDBObject(DBObject dbObject, @Nullable ProductionCompany model) {
+        DBObject companyDbObject = companyTranslator.toDBObject(model.getCompany());
+        TranslatorUtils.from(dbObject, "company", companyDbObject);
```
Comment (EmiEnache) [inline src/main/java/org/atlasapi/persistence/media/entity/ProductionCompanyTranslator.java:23]: Replace with COMPANY static final string

File: src/main/java/org/atlasapi/persistence/media/entity/ProductionCompanyTranslator.java [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    public ProductionCompany fromDBObject(DBObject dbObject) {
+        return fromDBObject(dbObject, new ProductionCompany());
+    }
+
+    @Override
+    public ProductionCompany fromDBObject(DBObject dbObject, @Nullable ProductionCompany model) {
+        if(model == null)
+            model = new ProductionCompany();
+
+        DBObject companyDbObj = TranslatorUtils.toDBObject(dbObject, "company");
```
Comment (EmiEnache) [inline src/main/java/org/atlasapi/persistence/media/entity/ProductionCompanyTranslator.java:40]: Replace with COMPANY static final string

File: src/main/java/org/atlasapi/persistence/media/entity/ProductionCompanyTranslator.java (hunk 1 above)
Comment (pawel-mtbcst) [reply src/main/java/org/atlasapi/persistence/media/entity/ProductionCompanyTranslator.java:33]: No, they're not :)

**Pull request closed without merging**